proto/storage/signature_integration.proto

```diff
@@ -12,6 +12,8 @@ message SignatureIntegration {
   CosignPublicKeyVerification cosign = 3;
 
   repeated CosignCertificateVerification cosign_certificates = 4;
+
+  TransparencyLogVerification transparency_log = 6;
```

Comment:
why not 5?

Reply:
Ah good catch thanks. I initially added another field for the TUF repository, but decided to move this to a later PR due to the complexity of the TUF client. Then forgot to update the numbers.